.gitignore

```diff
@@ -11,4 +11,4 @@ dist/
 
 # ducktape
 .ducktape
-*-test-results
+*results
```

Comment:
Not a major issue, but is this possibly too broad? It ignores anything ending in results. That's probably unusual for a single file (which will have an extension), but it seems like this could unintentionally match directories.

Any reason the line shouldn't be `results/` which would only match the results directory at the top level?